executor/infoschema_reader.go

```diff
@@ -2820,6 +2822,145 @@ func (e *memtableRetriever) setDataForRegionLabel(ctx sessionctx.Context) error
 	return nil
 }
 
+func (e *memtableRetriever) setDataFromPlacementRules(ctx context.Context, sctx sessionctx.Context, schemas []*model.DBInfo) error {
+	checker := privilege.GetPrivilegeManager(sctx)
+	is := sctx.GetInfoSchema().(infoschema.InfoSchema)
+	var rows [][]types.Datum
+
+	// Get global PLACEMENT POLICIES
+	// Currently no privileges needed for seeing global PLACEMENT POLICIES!
+	for _, policy := range is.AllPlacementPolicies() {
+		// Did we say that we would skip syntactic sugar?
+		// I proposed it to be transformed, xhebox proposed to keep it :)
+		// How about to do both, then one can see if syntactic sugar was used?
```

Comment:
I think if we keep this comment for now, it should be phrased as a statement:
"Currently we skip syntactic sugar because of XYZ. We might revisit this decision still in future"

My rationale is it is a little bit easier to parse without context.